nuxeo-services/nuxeo-platform-filemanager-core/src/main/java/org/nuxeo/ecm/platform/filemanager/service/extension/CSVZipImporter.java

```diff
@@ -27,9 +27,13 @@
 import java.io.Serializable;
 import java.text.ParseException;
 import java.text.SimpleDateFormat;
+import java.util.ArrayList;
```

Comment:
This and `import java.util.List;` are note used anymore since you now use streams.